### ISSUE TITLE
Fix http links to OSMF-run domains in translations

### ISSUE THREAD

**Issue by gravitystorm** [openstreetmap/openstreetmap-website#2392]
From https://trac.openstreetmap.org/ticket/5127

I found that many of the translations contain links to openstreetmap.org (and osmfoundation.org) sites via http. Although all the links in en.yml (the master translation file) have already been updated, there's around 796 strings in other languages where http is still being used.

```bash
$ grep -rn 'http:.*osmfoundation\.org' config/locales/ | wc -l
177
$ grep -rn 'http:.*openstreetmap\.org' config/locales/ | wc -l
619
```

These can be fixed fairly easily via searching for `http:` on translatewiki, and I did a bunch of them today in the en_GB and de locales. But it's time consuming, so if somebody else want to help, then that would be very useful.

Note that there are other domains in the translation files that use http. That's fine, please leave them for now. This is only in regards to *.openstreetmap.org and *.osmfoundation.org links.

**Comment by grischard** [comment]
Could this possibly be regex replaced?

**Comment by tomhughes** [comment]
Well that's not the problem...

Obviously we could bulk edit all the YAML files in the repository but then some poor fool at Translatewiki has to work out how to merge those edits with any edits that have been done on the web site since the last commit without losing anything.

That's why we don't allow local editing of locales other than english, which is the real problem here, no whether a bulk edit is theoretically possible - the en locale was bulk edited long ago so obviously it is possible ;-)

**Comment by gravitystorm** [comment]
It might be possible for a translatewiki sysadmin to fix these automatically, but since it's something we can fix ourselves I'd like to see if we can. Many hands make light work, etc, and I'd rather only bother the translatewiki sysadmins for things that we can't fix ourselves.

I also noticed while doing the first few dozen that many of the translated strings were out of date, beyond just the urls. For example, having OSM instead of OpenStreetMap in some en_GB translations, or missing updated text (e.g. #1686). I'm not sure why this is, perhaps the 'outdated' flag was lost somewhere along the line.

So, like autofixing tags in OSM, perhaps it's worth getting humans to glance at each issue rather than just running a script. But if nobody works through these in the next few weeks, then we can look at doing something more automated.

**Comment by simonpoole** [comment]
@gravitystorm did the strings in question turn up when you selected the "outdated" (or whatever its label in EN is) tab? I suspect that nobody really ever checks that.

**Comment by gravitystorm** [comment]
> @gravitystorm did the strings in question turn up when you selected the "outdated" (or whatever its label in EN is) tab? I suspect that nobody really ever checks that.

No, they didn't appear on that list for either en_GB nor de. I'm not sure what the problem there is, perhaps the outdated flag has been lost at some point? I haven't investigated that further since I'm not super familiar with the translatewiki interface and how to debug that.

**Comment by gravitystorm** [comment]
Slow progress so far:

```
$ grep -rn 'http:.*osmfoundation\.org' config/locales/ | wc -l
140
$ grep -rn 'http:.*openstreetmap\.org' config/locales/ | wc -l
512
```

Help with this would still be appreciated!

**Comment by AntonKhorev** [comment]
4 translations left

```
grep -rn 'http:.*osmfoundation\.org' config/locales/
config/locales/sq.yml:1302:      consider_pd_why_url: http://www.osmfoundation.org/wiki/License/Why_would_I_want_my_contributions_to_be_public_domain
config/locales/en-GB.yml:1846:      consider_pd_why_url: http://www.osmfoundation.org/wiki/License/Why_would_I_want_my_contributions_to_be_public_domain
config/locales/gd.yml:1651:      consider_pd_why_url: http://www.osmfoundation.org/wiki/License/Why_would_I_want_my_contributions_to_be_public_domain
config/locales/nn.yml:2408:      consider_pd_why_url: http://www.osmfoundation.org/wiki/License/Why_would_i_want_my_contributions_to_be_public_domain
```

**Comment by Dimitar5555** [comment]
> 4 translations left

I've just fixed them but there are about 120 of `http:.*openstreetmap\.org`.

**Comment by AntonKhorev** [comment]
Still present in en-GB.
